nodejs/webapi/server.js

```diff
@@ -0,0 +1,93 @@
+const express = require("express"); // import express
+const { swaggerSpecs, setupSwaggerUi } = require("./swagger/openapiservice");
+
+// JSDoc comments for api endpoints
+/**
+ * @swagger
+ * /weatherforecast:
+ *   get:
+ *    description: Use to request weather data
+ *    operationId: getWeatherData
+ *    tags:
+ *    - Weather
+ *    responses:
+ *     200:
+ *      description: A successful response
+ *      content:
+ *       application/json:
+ *        schema:
+ *         type: array
+ *         items:
+ *          type: object
+ *          properties:
+ *           date:
+ *            type: string
+ *            format: date
+ *            description: Date of the weather forecast
+ *            example: '2024-06-05'
+ *           temperatureC:
+ *            type: integer
+ *            description: Temperature in Celsius
+ *            example: 51
+ *           summary:
+ *            type: string
+ *            description: Weather summary
+ *            example: Cool
+ *           temperatureF:
+ *            type: integer
+ *            description: Temperature in Fahrenheit
+ *            example: 123
+ */
+
+const app = express();
+const port = process.env.PORT || 3030;
```

Comment:
If we set the `env.PORT` value to a different one like `8080`, it will cause the issue on the `swagger/openapiservice.js`